src/mapclient/tools/pmr/dialogs/register.py

```diff
@@ -27,13 +29,17 @@
 
 class PMRRegisterDialog(QtWidgets.QDialog):
 
-    def __init__(self, external_git, parent=None):
+    def __init__(self, workflow_widget, parent=None):
         super(PMRRegisterDialog, self).__init__(parent)
         self._ui = Ui_RegisterDialog()
         self._ui.setupUi(self)
 
+        self._workspace_widget = WorkspaceWidget(workflow_widget)
+        self._ui.workspaceTab.layout().addWidget(self._workspace_widget)
+
         pmr_info = PMR()
-        self._pmr_tool = PMRTool(pmr_info, external_git)
+        om = workflow_widget.model().optionsManager()
```

Comment:
If we are not storing the workflowwidget we should be passing in just what this dialog needs, in this  case whether the use external git option is True or False.